### DISCUSSION TITLE
World map error after upgrade to 1.0.1

### DISCUSSION THREAD

**Discussion by vialcollet** [apache/superset#18274] [Q&A / Help]
I updated from 0.38 to 1.0.1.
I performed db upgrade as expected.
Then when trying to display a world map chart (directly or from dashboard) it is not loading and this message is displayed:
```
Admin user does not exist
```
My only admin user has `bastien` as a username.

### Expected results

I am expecting the world map to be displayed eventhough there is no admin user with `admin` as a username.

### Actual results

Following message is displayed:

```
Unexpected error
Admin user does not exist. Please, check if test users are properly loaded (`superset load_test_users`).
```

#### Screenshots

![image](https://user-images.githubusercontent.com/4971086/107847885-802d5d00-6def-11eb-953c-a871e9f3d908.png)

#### How to reproduce the bug

1. Install superset
2. Create an admin with a specific username other than `admin`
3. Log on the UI
4. Create a World map graph

### Environment

(please complete the following information):

- superset version: 1.0.1
- python version: 3.8.7
- node.js version: 12

### Checklist

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x ] I have reproduced the issue with at least the latest released version of superset.
- [x ] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

I have workaround the issue by adding an admin user with `admin` as a username.

**Comment by vialcollet** [comment]
It's worth mentioning that this was working as expected in 0.38

**Comment by TheRum** [comment]
I'm also facing the same issue after upgrading to Superset 1.0.1.

**Comment by S-eb** [comment]
facing similar issues.upgrading from 0.38 to 1.0.1
creating an admin account named 'admin' seems to be a valid workaround

**Comment by TheRum** [comment]
I have upgraded Superset to 1.1.0 and still facing this issue .
